### PR TITLE
Changelog for 1.1.8

### PR DESCRIPTION
**Pull request by gtardif**

### CODE REVIEW

**Review by mat007 — APPROVED** [review]

File: ui/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "@types/react-dom": "^17.0.2",
     "@typescript-eslint/eslint-plugin": "^5.36.0",
     "@typescript-eslint/parser": "^5.36.0",
-    "eslint": "~9.6.0",
+    "eslint": "~8.22.0",
```
Comment (mat007) [inline ui/package.json:39]: Is this change intentional in a PR that seems only about updating the changelog?

Comment (gtardif) [reply ui/package.json:39]: Oops, nope, a left over file, too many context switching. Thx
